nixos/release.nix

```diff
@@ -92,15 +92,33 @@ let
   }).config));
 
 
+  cleanConfig = import ./lib/eval-config.nix {
```

Comment:
Yes, though that `scrubDerivations` might have too much of an impact on evaluation speed. Which is too bad because the idea of having attribute names in the manual instead of store paths is pretty nice.